pkgs/servers/nosql/ferretdb/default.nix

```diff
@@ -6,21 +6,21 @@
 
 buildGo121Module rec {
```

Comment:
go 1.21 is default, this can be changed to `buildGoModule`

Reply:
```suggestion
buildGoModule rec {
```